openfl-docker/Dockerfile.base

```diff
@@ -53,9 +111,9 @@ WORKDIR /openfl
 COPY . .
 
 # Install pip
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1
 RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python get-pip.py && rm -rf get-pip.py
-RUN pip install --no-cache-dir . 
+RUN pip install --no-cache-dir . && \
+    pip uninstall -y setuptools
```

Comment:
`setuptools` is a required package: https://github.com/securefederatedai/openfl/actions/runs/8137042934/job/22234556956?pr=937#step:5:12376